hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -175,6 +175,12 @@
       sw_images: ["sw/device/tests/aes_test:1"]
       en_run_modes: ["sw_test_mode"]
     }
+    {
+      name: chip_dif_gpio_smoketest
```

Comment:
I think it might be worth adding a TODO here to write a custom SV test sequence that checks the GPIO values seen at the chip periphery.

Reply:
This is now done in ffb2720.

Reply:
Thanks @sriyerg!